xbmc/cores/AudioEngine/Sinks/pipewire/PipewireNode.cpp

```diff
@@ -40,7 +40,7 @@ void CPipewireNode::EnumerateFormats()
   for (uint32_t param = 0; param < m_info->n_params; param++)
   {
     if (m_info->params[param].id == SPA_PARAM_EnumFormat)
-      pw_node_enum_params(m_proxy.get(), 0, m_info->params[param].id, 0, 0, NULL);
+      pw_node_enum_params((struct pw_node*)m_proxy.get(), 0, m_info->params[param].id, 0, 0, NULL);
```

Comment:
Can you use a C++ style cast here as per coding guidelines?

Reply:
Changed it and the built went fine for me, hope this is correct.